Adding descriptions to classes and concepts in the dco ontology

mrpatrickwest opened the pull request.

Commit:
Adding descriptions to classes and properties of dco ontology

First pass at adding descriptive information about all of the classes
and properties of the DCO ontology.

Commit:
Corrected syntax problems after adding descriptions

When adding descriptions added at the end of a class or concept and
needed to change the terminating character, a dot, to a semicolon. Also,
some of the older descriptions spanned multiple lines but used the
incorrect formatting in the n3.

xgmachina commented:
@mrpatrickwest Can you check the rdfs:comments of dco:assessURL and dco:downloadURL? I think they need a switch.

zednis commented:
Why do all the comments have type XMLLiteral?

zednis commented:
There are several multi-line comments with unnecessary newlines (e.g. hasGeoSample, IGSN, DataType)

Commit:
data type classes and properties descriptions updated

After speaking with Marshall about data types, updated the labels and
descriptions of the different classes and properties that make up the
dco:DataType part of the ontology.
